src/Tizen.Account.AccountManager/Tizen.Account.AccountManager/AccountService.cs

```diff
@@ -108,7 +108,6 @@ public static IEnumerable<Account> GetAccountsAsync()
             {
                 Account account = new Account(new SafeAccountHandle(data, true));
                 values.Add(account.AccountId);
-                account.Dispose();
```

Comment:
Why do you remove Dispose call?

Reply:
Actually, SafeAccountHandle has no ownership
(data is not vaild outside of callback)

so it should be like below
``` c#
new SafeAccountHandle(data, false);
```